### ISSUE TITLE
add plone.app.discussion docs

### ISSUE THREAD

**Issue by svx** [plone/documentation#21]
move them from old kb

**Comment by svx** [comment]
@polyester @tisto ping

**Comment by svx** [comment]
@polyester @tisto ping ! 

You can also do that next to the pool ! :)

**Comment by tisto** [comment]
@svx pong. What can I do to help?

**Comment by svx** [comment]
@tisto talk with @polyester about this open ticket here :)

- is that ticket valid for Plone 5 ?
- if yes what needs to be done to get it solved and the docs of plone.app.discussion into docs.plone.org ?

Last thing what @polyester said, 3 years ago: 

> no, not yet. The documentation pulls in lots of stuff from the /src directory, and uses extra modules. Will have to work with tisto to see how to best handle this one.

:)

**Comment by tisto** [comment]
Yeah. It is true that we use "inline" docs in plone.app.discussion. I kind of like the idea that the docs are next to the python file and sphinx allows you to do that. We have to decide if we either fetch the src directory or refactor the docs and move stuff into docs.

**Comment by stevepiercy** [comment]
We should import plone.app.discussion similar to how we do plone.api and plone.restapi with a git submodule. How to guide: https://6.docs.plone.org/contributing/documentation/admins.html#importing-external-docs-with-submodules
